src/dex-ui-components/base/Icons/CheckCircleIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import { createIcon } from "@chakra-ui/react";
+
+export const CheckCircleIcon = createIcon({
+  displayName: "CheckCircleIcon",
+  viewBox: "0 0 20 20",
+  path: (
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M10 17a7 7 0 1 0 0-14 7 7 0 0 0 0 14Zm3.244-8.131a.875.875 0 0 
```

Comment:
https://jakearchibald.github.io/svgomg/ was used to reduce the length of the <path> `d` attributes.